recipes/clhep/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake ../CLHEP/ -DCMAKE_SHARED_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib" -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}
```

Comment:
You can remove `-DCMAKE_SHARED_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib"` and `-DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}`